### DISCUSSION TITLE
Poblems binging it to work

### DISCUSSION THREAD

**Discussion by beldowsk** [ozel/DIY_particle_detector#50] [Q&A] [answered]
Hello,

I have worked through the troubleshooting points, unfortunately without finding an error.

[2023_02_20 Fotos.zip](https://github.com/ozel/DIY_particle_detector/files/10787077/2023_02_20.Fotos.zip)

The assembled board has an internal resistance of 9.71 kOhm. A charged 9.6V battery is installed.

A "Quantum Pendant" serves as a test object. A radioactive pendant from the esoteric field. With a GammaScout it delivers the following measured values:
α + β + γ 49.6 cpm - 19 cpm (background = 30.6 cpm
β + γ 44.00 cpm - 19 cpm (background) = 25.0 cpm
γ 25.4 cpm - 17.1 (background) = 8.3 cpm

I use an onboard chip in my desktop as the AD converter. A Realtek S1220A (32 bit, 192 kHz, 30 dB input gain).

In the online oscilloscope https://ozel.github.io/DIY_particle_detector/data_recording_software/webGui/ it just fidgets around the zero line.

![Online_Oszilloskop_DIY_ohne_Strom](https://user-images.githubusercontent.com/125914157/220179452-f70444e7-5496-45cb-8520-18f7b0210448.jpg)
![Online_Oszilloskop_DIY_mit_Strom](https://user-images.githubusercontent.com/125914157/220179526-a63bd6bc-5426-4bd1-819d-288707f7fea0.jpg)

WaveWatch ran under processing 4th images first without and than with power supply.
![WaveWatch_DIY_ohne_Strom](https://user-images.githubusercontent.com/125914157/220179596-4368635f-a6da-4843-99f7-779dd0e09d2c.jpg)
![WaveWatch_DIY_mit_Strom](https://user-images.githubusercontent.com/125914157/220179606-2993764e-ef95-4826-8151-76c78ff81994.jpg)

The Soundcard Scope first without and than with power.
![Scope_DIY_ohne_Strom](https://user-images.githubusercontent.com/125914157/220179695-017f7e33-4d2f-446c-8cb3-fcb4fb3c8a13.jpg)
![Scope_DIY_mit_Strom](https://user-images.githubusercontent.com/125914157/220179699-8ef13b77-4931-4f7e-bc4d-537732c76f3e.jpg)

Greetings Andreas

**Comment by SteveF161** [comment, marked as answer]
Two things which I hope are useful:
1. A sophisticated ADC and soundcard are actually a hindrance as noise suppression interferes with the signal. I used a very cheap (< 10 GBP) , unsophisticated USB device and it worked beautifully. See my posts from 2020
2. Your cable looks like a cheap phono cable with several interposed adapters resulting in poor shielding. I found my device was very sensitive to electrical noise and a properly shielded connection is essential.

**Reply by beldowsk** [reply]
Hello,

when using the onboard soundcard with +30dB input gain, I must have caught so much noise that the impulses were lost in it. Now I use an external steinberg UR22C (32 bit, 192000 Hz). 
I replaced the many adapters with a short BNC to jack cable.

![2023_02_22_DIY_on_UR22C](https://user-images.githubusercontent.com/125914157/221350429-9df37b09-a8ef-4100-ad85-07cee4366221.jpg)

Yesterday morning I could see 10 nice strong pulses within one hour (soundcard scope trigger -50 mV). I had to set the input sensitivity of the UR22C to about 65%, otherwise the pulses were cut off.
 
![Scope_UR22C_DIY_mit_Strom_Impuls17](https://user-images.githubusercontent.com/125914157/221350437-c3949e82-62e1-4a8a-9588-48892ca014ad.jpg)

Unfortunately, this did not work yesterday afternoon?! Is it possible that my battery is already not full enough?
With a full battery this morning I had to go up to 85% input sensitivity of the UR22C to be able to see 10 impulses in the hour with a trigger of -10 mV (soundcard scope).
Did I see anything other than impulses from radioactivity yesterday?
Is the above impulse from the transit time (time axis 1 ms/div) OK for the DIY Particle Detector?

Greetings Andreas

**Reply by ozel** [reply]
Dear Andreas, 
first of all, congratulations! 

That is a picture perfect pulse you recorded now. Anything of _similar shape_ must stem from real ionisation events inside the diode. 
If you hit the metal enclosure, you may observe damped oscillations, which is normal for such sensitive electrometers as the detector circuit.  But those 'microphonics' type of signals are are always continuous (rising/falling amplitudes) and never have a steep downward edge followed by one long asymmetrical pulse with overshoot.
You can find reference pulses here: https://github.com/ozel/DIY_particle_detector/wiki/Oscilloscope-Measurements

Since the signal over noise ratio seems quite high in your case, you must have caught an alpha particle of high energy. You can place the trigger much close to the zero level for catching some lower energy alphas and betas (plus few gammas). Just make sure you only trigger on steep downward edges and nothing else (electronic noise looks more symmetrical and linear than a real pulse, see the wiki page linked above).

Those pendants exhibit varying levels of radioactivity due to different thorium oxide concentrations. If your nice oscilloscope software features a "persistance" capture/acquisition mode, you should be able to record a full beta/alpha spectrum like in figure 9a here: https://www.mdpi.com/1424-8220/19/19/4264/htm. Then you can determine the beta/alpha threshold for your amplifier/soundcard setup just by observing pulses over a whole night/day. A thick band of overlapped pulses close to zero will relate to beta/gamma pulses, anything stronger must be from an occasional alpha particle.

The batteries hold up a few hours and recover for a few minutes if the device was of for some time - that can be misleading! See [the corresponding page on batteries in the wiki](https://github.com/ozel/DIY_particle_detector/wiki/Batteries). 
I really recommend the good old NiMH accumulators, there's a 9.6 V version which is perfect for the alpha detector.

In your pictures above (awesome construction by the way!) attached to your initial post, the audio chinch cable was clearly the problem as @SteveF161 pointed out. The signal output must be shielded, coax is best, shielded microphone cables work as well.

May I copy some of your detector pictures into the gallery page of the wiki?

**Reply by beldowsk** [reply]
Yes, they are allowed on the wiki. Should I send more? Sondcard Scope always saves the original scope image, a black and white image and a csv file.
![Scope_UR22C_DIY_mit_Strom_Impuls16](https://user-images.githubusercontent.com/125914157/221376410-0abfbe28-93c6-4c22-aec4-69a5a1958922.jpg)
![Scope_UR22C_DIY_mit_Strom_Impuls16_bw](https://user-images.githubusercontent.com/125914157/221376411-6d8d2fb5-9bd6-4411-a694-42c378a8317e.jpg)
[Scope_UR22C_DIY_mit_Strom_Impuls16.csv](https://github.com/ozel/DIY_particle_detector/files/10832126/Scope_UR22C_DIY_mit_Strom_Impuls16.csv)

**Reply by beldowsk** [reply]
Why don't I get such images with the DIY Particle Detector or the WaveWatch? Is it because of the 32 bit resolution? The Particle Detector shows "sampling: 2048@192 kHz" and the WaveWatch is set to "sampel_freq = 192000". Then it can't be because of the sampling rate?!

**Reply by beldowsk** [reply]
Or is it the special driver
![UR22C_driver_01](https://user-images.githubusercontent.com/125914157/221377674-0c160674-b025-4165-852c-5316858cb8b0.jpg)
![UR22C_driver_02](https://user-images.githubusercontent.com/125914157/221377677-6dd3b881-ac4e-41a5-8656-0110b7d89b1d.jpg)
I have 3 Modes: Low Latency, Standard, Stable

**Reply by beldowsk** [reply]
![Scope_UR22C_090_DIY_Impuls20_080mV](https://user-images.githubusercontent.com/125914157/221398871-832c6c45-25d3-43f0-b94f-329c179678a7.jpg)
90% input sensitivity UR22C, 75mV/Div, Trigger -80mV, min. -339mV, max. 117 mV, tmax-tmin=2,250ms

![Scope_UR22C_095_DIY_Impuls25_400mV](https://user-images.githubusercontent.com/125914157/221398900-9c4bf4c2-5791-4ea1-b200-61d2a015975c.jpg)
95% input sensitivity UR22C, 200mV/Div, Trigger -400mV, min. -751mV, max. 262 mV, tmax-tmin=2,182ms

![Scope_UR22C_100_DIY_Impuls23_150mV](https://user-images.githubusercontent.com/125914157/221398935-5e9cab45-45b1-4acd-b3c2-3225aca0c71c.jpg)
Full input sensitivity UR22C (100%), 250mV/Div, Trigger -150mV

**Reply by ozel** [reply]
sorry, I never answered to your last point. The sample rate specified in the WaveWatch sketch is misleading. The underlying java/Processing.org code does not actually apply the sample rate as parameter to the sound device, it opens the devices always with their default rate. I also found this out only after some time trying. 
Most likely WaveWatch doesn't open the correct audio device (maybe only the internal sound card), so another device needs to be specified in the code or you have to mark the Steinberg device as default sound device in Windows or something like that.

Anyway, WaveWatch does not offer anything more advanced than your fancy audio oscilloscope app, rather less! For counting of pulses per time the webpage pulse recorder can be used or even better, the python scripts in this repository.
On Windows, you can also try the Theremino MCA https://github.com/ozel/DIY_particle_detector/wiki/Software#graphical-software
Even if you have not built the alpha spectrometer (where one needs an MCA-like software), Theremino still displays raw count rates of pulses if I remember well.

**Reply by ozel** [reply]
I am surprised that the browser-based Pulse Recorder actually showed 192 kHz because until recently Firefox limited the microphone sample rates to 48 kHz. It's always a good idea to try a recent Google Chrome version as well, it has by far the best and most complete javascript audio support.
Note that you need to change the red trigger line by pressing the down arrow key until you get out of the noise floor such that it triggers only on pulses.

**Reply by Startrekhumansmaybe** [reply]
hey hey hey Mr. STEVE how's it going my main man, got a question for you. 
We are using an alpha particle detector from the CERN kit. We were using the website and it was working for about 30 seconds and was properly reading the electrons and the alpha particles. But then it suddenly stopped working and we can't get it to work anymore. Any ideas on what to do, or have any solutions?\

Thanks Mr. STEVE (the human) in advance.
